openapi.yaml

```diff
@@ -0,0 +1,675 @@
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AsyncOperation'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '410':
+          description: Gone
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '422':
+          description: Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /v2/service_instances/{instance_id}/last_operation:
+    get:
+      summary: last requested operation state for service instance
+      tags:
+        - ServiceInstances
+      operationId: serviceInstance.lastOperation.get
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - name: instance_id
+          in: path
+          description: instance id of instance to find last operation applied to it
+          required: true
+          schema:
+            type: string
+        - name: service_id
+          in: query
+          description: id of the service being used
+          schema:
+            type: string
+        - name: plan_id
+          in: query
+          description: id of the plan being used
+          schema:
+            type: string
+        - name: operation
+          in: query
+          description: a provided identifier for the operation
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInstanceLastOperation'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+        '410':
+          description: Gone
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Object'
+
+  /v2/service_instances/{instance_id}/service_bindings/{binding_id}:
+    put:
+      summary: generation of a service binding
+      tags:
+        - ServiceBindings
+      operationId: serviceBinding.binding
+      parameters:
+        - $ref: '#/components/parameters/APIVersion'
+        - $ref: '#/components/parameters/OriginatingIdentity'
+        - name: instance_id
+          in: path
+          description: instance id being bound
+          required: true
+          schema:
+            type: string
+        - name: binding_id
+          in: path
+          description: binding id being bound
```

Comment:
Suggestion: `binding id of binding to create` (or something else - I'm not sure the current description makes sense)